operations/services.md

```diff
@@ -143,62 +165,50 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 
 #### Time unit period
 
-* `Hour`
 * `Day`
 * `Month`
-* ...
+* `Hour`
 
 #### Additional service data
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Promotions` | [Promotions](#promotions) | required | Promotions of the service. |
 
-#### Promotions
-
-| Property | Type | Contract | Description |
-| :-- | :-- | :-- | :-- |
-| `BeforeCheckIn` | boolean | required | Whether it can be promoted before check-in. |
-| `AfterCheckIn` | boolean | required | Whether it can be promoted after check-in. |
-| `DuringStay` | boolean | required | Whether it can be promoted during stay. |
-| `BeforeCheckOut` | boolean | required | Whether it can be promoted before check-out. |
-| `AfterCheckOut` | boolean | required | Whether it can be promoted after check-out. |
-| `DuringCheckOut` | boolean | required | Whether it can be promoted during check-out. | 
-
 ## Get service availability (ver 2024-01-22)
 
 > ### Restricted!
 > This operation is currently in beta-test and as such it is subject to change.
-
-Returns selected availability and occupancy metrics of a bookable service for a specified time interval, similar to [the availability & occupancy report](https://help.mews.com/s/article/Availability-Occupancy-report). Availability will be returned for all service [time units] (../guidelines/time-units.md) that the specified time interval intersects. So, for example, an interval `1st Jan 23:00 UTC - 1st Jan 23:00 UTC` will result in one time unit for `2nd Jan`, while Interval `1st Jan 23:00 UTC - 2nd Jan 23:00 UTC` will result in two time units for `2nd Jan` and `3rd Jan` (assuming a time unit period of "Day"). UTC timestamps must correspond to the start boundary of a [time unit] (../guidelines/time-units.md), e.g. 00:00 converted to UTC for a time unit of "Day". Other timestamps are not permitted. The __maximum size of time interval__ depends on the service's [time unit](../guidelines/time-units.md): 367 hours if hours, 367 days if days, or 24 months if months.
+Returns selected availability and occupancy metrics of a bookable service for a specified time interval, similar to [the availability & occupancy report](https://help.mews.com/s/article/Availability-Occupancy-report). Availability will be returned for all service [time units](services.md#time-unit) that the specified time interval intersects. So, for example, an interval `1st Jan 23:00 UTC - 1st Jan 23:00 UTC` will result in one time unit for `2nd Jan`, while Interval `1st Jan 23:00 UTC - 2nd Jan 23:00 UTC` will result in two time units for `2nd Jan` and `3rd Jan` (assuming a time unit period of "Day"). UTC timestamps must correspond to the start boundary of a [time unit](services.md#time-unit), e.g. 00:00 converted to UTC for a time unit of "Day". Other timestamps are not permitted. The __maximum size of time interval__ depends on the service's [time unit](services.md#time-unit): 367 hours if hours, 367 days if days, or 24 months if months.
```

Comment:
Now `../concepts/time-units.md` :-) As a general point, going forwards we should try to avoid linking outside of the API Reference, in other words linking anywhere except to other API Operations pages.